src/components/alert/index.js

```diff
@@ -6,16 +6,28 @@ import { ReactComponent as Loading } from '../../images/loading.svg';
 
 import './index.css';
 
-const Alert = ({ text, loading, error }) => (
-  <div className='alert' data-error={error}>
-    {loading && <Loading />}
-    {!loading && <AlertIcon />}
-    <span>{text}</span>
-  </div>
-);
+const Alert = ({ status = '', subject = '', className = '' }) => {
+  let text = status;
+  if (status === 'loading')
+    text = 'Loading ' + subject;
+  if (status === 'empty')
```

Comment:
`else if` on this line (and line 15) would be a little more efficient, because these three `if`s are exclusive.